scripts/arc/documentation/arcdocs.md

```diff
@@ -0,0 +1,270 @@
+
+# Enable monitoring of Azure Arc enabled Kubernetes cluster
+
+[Azure Monitor for containers](https://docs.microsoft.com/azure/azure-monitor/insights/container-insights-overview) provides rich monitoring experience for Azure Arc enabled Kubernetes clusters.
+
+> **Retirement Notice:** The script-based version of Azure Monitor for Arc enabled Kubernetes (preview) is being replaced with the extension model. Users will have until June 2021 to upgrade to the extension model. Details on how to enable monitoring via the extension model are detailed below.
+
+This article describes how to [create](#create-azure-monitor-extension-instance), [display configuration settings](#display-configuration-settings) and [delete](#delete-extension-instance) the Azure Monitor for containers extension.
+
+## Supported configurations
+
+Azure Monitor for containers supports monitoring Azure Arc enabled Kubernetes (preview) as described in the [Overview](container-insights-overview.md) article, except for the following features:
+
+- Live Data (preview)
+- Users are not required to have [Owner](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#owner) permissions to [enable metrics](https://docs.microsoft.com/en-us/azure/azure-monitor/containers/container-insights-update-metrics)
+
+
+The following is officially supported with Azure Monitor for containers:
+
+- Versions of Kubernetes and support policy are the same as versions of Azure Arc
+
+- The following container runtimes are supported: Docker, Moby, and CRI compatible runtimes such CRI-O and ContainerD.
+
+- Linux OS release for master and worker nodes supported are: Ubuntu (18.04 LTS and 16.04 LTS).
```

Comment:
We need to align to Arc K8s supported OS distros as well since we will be doing validation and release as per for Arc K8s.